.github/workflows/coverage.yml

```diff
@@ -0,0 +1,26 @@
+name: Coverage report
+on:
+  push:
+  pull_request:
+    branches: ['main']
+jobs:
+  coverage:
+    name: Coverage
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.8'
+      - name: Install dependencies
+        run: pip install tox coveragepy-lcov
+      - name: Run coverage report
+        run: tox -ecoverage
```

Comment:
I think this will miss coverage of tests that need Aer, or other "downstream" plugins of Terra - we don't (can't) specify them in `requirements-dev.txt`, so they don't get pulled into the `tox` environments.  I suppose it's not _imperative_ that we include those tests, but it would be nicer for our stats if we could!

Reply:
Yeah, we should just add qiskit-aer and anything else to the coverage tox job definition.